cluster/docker-cluster/kube-config/master-multi.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
I think that's fine, but I'm not sure if we should copy this file here. We already have this in ```cluster/images/hyperkube```. Maybe it's enough to point users there?